conda/conda-build/meta.yaml

```diff
@@ -72,6 +72,8 @@ requirements:
     - cuda-nvcc ={{ cuda_version }}
 {% endif %}
   host:
+    # the nvcc requirement is necessary because it contains crt/host_config.h used by cuda runtime. This is a packaging bug that has been reported.P
```

Comment:
P?

Reply:
I was trying to hit a vs code keyboard shortcut, but I hit a wrong key and ended typing in P I guess. :)